### PR TITLE
Discord social symbol

### PR DESCRIPTION
**Pull request by mefuller**
Add Discord social icon

### CODE REVIEW

**Comment by stephanlachnit** [comment]
Interesting, the CI fails reliably

**Comment by mefuller** [comment]
Ready for review - closes #111

**Comment by mefuller** [comment]
It still doesn't all fit even with the extended footskip - I know casual is the default, but there's so many social icons now that perhaps making the template/default classic style would be preferable (last commit)
Thoughts?

**Comment by stephanlachnit** [comment]
> It still doesn't all fit even with the extended footskip - I know casual is the default, but there's so many social icons now that perhaps making the template/default classic style would be preferable (last commit) Thoughts?

Yes I think this is probably better - while it looks hilarious IMHO with that many social symbols, it's better for testing

**Comment by mefuller** [comment]
@stephanlachnit in that case, whenever you are finished reviewing, please go ahead and merge

**Review by koppor — COMMENTED** [review]
`template.tex` got an update of the style from `casual` to `classiv`, which is IMHO not really intened, is it?

File: template.tex [hunk 1]
```diff
@@ -9,14 +9,14 @@
 \documentclass[11pt,a4paper,sans]{moderncv}        % possible options include font size ('10pt', '11pt' and '12pt'), paper size ('a4paper', 'letterpaper', 'a5paper', 'legalpaper', 'executivepaper' and 'landscape') and font family ('sans' and 'roman')
 
 % moderncv themes
-\moderncvstyle{casual}                             % style options are 'casual' (default), 'classic', 'banking', 'oldstyle' and 'fancy'
+\moderncvstyle{classic}                             % style options are 'casual' (default), 'classic', 'banking', 'oldstyle' and 'fancy'
```
[the remark this answers is not in the document]
Comment (stephanlachnit) [reply template.tex:12]: It is actually intended, as otherwise not all social icons are visible without an insane footskip

**Pull request merged**